doc/configuration/authentication_modes.rst

```diff
@@ -41,113 +41,178 @@ modes in more detail.
 ``authenticate`` mode
 ---------------------
 
- * The user enters a OTP PIN along with an OTP value.
- * The plugin sends a request to the ``/validate/check`` endpoint:
+.. uml::
+  :width: 500
 
-   .. code-block:: text
+  Service -> PrivacyIDEA: POST /validate/check
```

Comment:
You are adding the word *Service* here.
Maybe we should add some notes, what the *Service* is: that *Service* is the application that gets the 2FA protection by talking to privacyIDEA. I currently have no perfect wording myself.

Please write privacyIDEA with a lower "p".